content/en/blog/2025/ambient-multicluster/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Introducing multicluster support for ambient mode
+description: Introducing multicluster support for ambient mode.
+date: 2025-08-04
+attribution: Jackie Maertens (Microsoft), Keith Mattix (Microsoft), Mikhail Krinkin (Microsoft), Steven Jin (Microsoft)
+keywords: [ambient,multicluster]
+---
+
+Multicluster has been one of the most requested ambient features — and as of Istio 1.27, it's now available.
+We sought to capture the benefits and avoid the complications of multicluster architectures using the same modular design that ambient users love.
+While still in alpha, this release delivers the core functionality of a multicluster mesh and lays the groundwork for a full feature set in upcoming releases.
+
+## Multicluster's Many Benefits (and Challenges)
+
+Multicluster architectures increase outage resilience, shrink the blast radii,
+ease adoption of data residence policies, and simplify cost tracking.
+That said, integrating multiple clusters poses connectivity, security, and operation hurdles.
+
+In a single Kubernetes cluster, every pod can directly connect to another pod via a unique pod IP or service VIP.
+We lose these guarantees when we start thinking of multicluster architectures.
+IP address spaces of different clusters might overlap.
+Even if they didn't, nodes in one cluster would not know how to route traffic from one cluster to another.
+
+Establishing cross-cluster connectivity also presents security challenges.
+Cross-cluster connectivity means that pod-to-pod traffic can leave cluster boundaries -- and that pods may accept connections from outside the cluster.
```

Comment:
We don't allow users to write policies based on the cluster the traffic resides in so I'd rephrase to talk about the complexity of deriving an identity model across clusters

Reply:
I'm interpreting the sentence as "Cross-cluster connectivity means that traffic is (often crossing the cluster) without going through an ingress gateway."

I don't really want to highlight differentiate the difference between ingress and e/w gateways. They are both proxies that can apply some sort identity verification and route traffic to their destinations within the cluster. I don't think that, from an identity point of view, they are inherently different (though in practice they are used for different purposes and configured with very different policies). But I do like more explicitly mentioning identity enforcement at the edge of the cluster. I would just frame it in terms of the attacker